metrics/anls/compute_score.py

```diff
@@ -0,0 +1,32 @@
+from Levenshtein import ratio
+
+
+def compute_score(predictions, ground_truths):
+    theta = 0.5
+    anls_score = 0
+    for qid, prediction in predictions.items():
+        max_value = 0
+        if qid in ground_truths:
+            for x in ground_truths[qid]:
+                nl = ratio(prediction, x)
+                if nl < theta:
+                    score = 1 - nl
+                    if score > max_value:
+                        max_value = score
+            anls_score += max_value
+
+    return anls_score
```

Comment:
I think we could move that to the main file and drop the `compute_score.py`.

Reply:
okay. Moved to main file and deleted compute_score.py file